### DISCUSSION TITLE
Refecth queries at an interval

### DISCUSSION THREAD

**Discussion by cubecleveland** [TanStack/query#2217] [General] [answered]
Hello,

I have a component that is mounted once per dom router in my applciation.
On mount  use useQuery to fetch the data....
Can i have the query refecthed itsled without any thing happening... at a specific interval....?

**Comment by TkDodo** [comment, marked as answer]
there is the refetchInterval option for that

**Reply by cubecleveland** [reply]
Perfect thank you.!
